nss-day-cohort-46/reactive-nutshell-talisman-of-havocing
# Sc task list

# Description
TaskList should render, when checkbox for isComplete is changed, should toggle bool in db
You may have to change completionDate in db, format was incorrect.
copy/paste this instead:  "2021-02-15T09:00"
## Type of change


# Testing Instructions
git fetch --all
git checkout sc-TaskList

click Tasks in nav bar
make sure To-Do list renders, and isComplete bool changes when checkbox is changed

# Checklist:
